pymc_bart/bart.py

```diff
@@ -82,18 +82,20 @@ class BART(Distribution):
         How the leaf_node values are computed. Available options are ``constant``, ``linear`` or
         ``mix``. Defaults to ``constant``.
     alpha : float
-        Controls the prior probability over the depth of the trees. Should be in the (0, 1) interval.
+        Controls the prior probability over the depth of the trees.
+        Should be in the (0, 1) interval.
     beta : float
-        Controls the prior probability over the number of leaves of the trees. Should be greater or bigger than 0.
+        Controls the prior probability over the number of leaves of the trees.
+        Should be greater or bigger than 0.
```

Comment:
```suggestion
        Should be positive.
```

Reply:
ok! Fixed in https://github.com/pymc-devs/pymc-bart/pull/95/commits/0653d3547b3ba575e47a14d63cc8bc972e7747ad